pkg/cloudclient/aws/private.go

```diff
@@ -378,7 +380,11 @@ func (c *Client) validateEgress(ctx context.Context, vpcSubnetID, cloudImageID s
 	}
 
 	c.logger.Info(ctx, "Gathering and parsing console log output...")
-	c.findUnreachableEndpoints(ctx, instanceID)
+	err = c.findUnreachableEndpoints(ctx, instanceID)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
c.logger.Error(ctx, err.Error())

Reply:
tbh, we don't even need to print out the err here as it'd be printed in the context of `output` anyway